packages/ERTP/src/issuerKit.js

```diff
@@ -85,9 +94,52 @@ const makeIssuerKit = (
     displayInfo: cleanDisplayInfo,
   });
 };
+harden(makeDurableIssuerKit);
 
-harden(makeIssuerKit);
-
-export { makeIssuerKit };
+/** @typedef {ReturnType<typeof makeDurableIssuerKit>} IssuerKit */
 
-/** @typedef {ReturnType<typeof makeIssuerKit>} IssuerKit */
+/**
+ * @template {AssetKind} K
+ * The allegedName becomes part of the brand in asset descriptions. The
+ * allegedName doesn't have to be a string, but it will only be used for
+ * its value. The allegedName is useful for debugging and double-checking
+ * assumptions, but should not be trusted.
+ *
+ * The assetKind will be used to import a specific mathHelpers
+ * from the mathHelpers library. For example, natMathHelpers, the
+ * default, is used for basic fungible tokens.
+ *
+ *  `displayInfo` gives information to the UI on how to display the amount.
+ *
+ * @param {string} allegedName
+ * @param {K} [assetKind=AssetKind.NAT]
+ * @param {AdditionalDisplayInfo} [displayInfo={}]
+ * @param {ShutdownWithFailure=} optShutdownWithFailure If this issuer fails
+ * in the middle of an atomic action (which btw should never happen), it
+ * potentially leaves its ledger in a corrupted state. If this function was
+ * provided, then the failed atomic action will call it, so that some
+ * larger unit of computation, like the enclosing vat, can be shutdown
+ * before anything else is corrupted by that corrupted state.
+ * See https://github.com/Agoric/agoric-sdk/issues/3434
+ * @returns {{
+ *  mint: Mint<K>,
+ *  issuer: Issuer<K>,
+ *  brand: Brand<K>,
+ *  displayInfo: DisplayInfo,
+ * }}
+ */
+export const makeIssuerKit = (
```

Comment:
why retain the non-durable version?

Reply:
Tons of existing callers of the existing APIs that I am unprepared to break or fix.

Reply:
>  so it's not the original non-durable function

Yeah, I meant retaining the _availability_ of a non-durable version.

> unprepared to break or fix

what would break if there was one `makeIssuerKit()` which returned a durable one?

Reply:
There are 350 calls to `makeIssuerKit` in agoric-sdk, spread across 107 files. Most of those are tests that don't care about durability, so we don't want to require that those be refactored to pass in a baggage argument.

When we do want to make an *upgradable* issuerKit, we need to call a maker to which we do pass a baggage argument. But either way, there is only one underlying implementation of the issuerKit, and that one implementation makes a durable issuer kit.

Because `makeIssuerKit` is not given a baggage, the issuerKit it gives you back is durable but not upgradable.